src/Stratis.Bitcoin.IntegrationTests/BlockStoreTests.cs

```diff
@@ -245,6 +245,7 @@ public void BlockStoreCanReorg()
 
                 // remove node 2
                 stratisNodeSync.CreateRPCClient().RemoveNode(stratisNode2.Endpoint);
+                TestHelper.WaitLoop(() => !TestHelper.IsNodeConnected(stratisNode2));
```

Comment:
What happens if the node doesn't disconnect at all?
Would it be a good idea to put a one-minute timeout with an error?

Reply:
Yeah The method WaitLoop will throw after a timeout